.github/workflows/release.yml

```
@@ -0,0 +1,68 @@
+name: Bump release version
+on:
+  pull_request:
+    branches: [rolling]
+    types:
+      - closed
+
+permissions:
+  contents: write
+  pull-requests: write
+  repository-projects: write
+
+concurrency: production
+
+jobs:
+  build:
+    name: bump tag version and release  
+    if: github.event.pull_request.merged == true 
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.sha }}
+          fetch-depth: 0
+      - name: Bump version and push tag
+        id: version-bump
+        uses: hennejg/github-tag-action@v4.3.1
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          release_branches: rolling
+      - name: create branch 
+        uses: peterjgrainger/action-create-branch@v2.2.0
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          branch: update-from-${{ github.sha }}
+      - name: update cargo.toml
+        run: |
+          appversion=$(echo "${{ steps.version-bump.outputs.new_tag }}" | sed 's/[v]//')
+          sed -i -e "s/^version = .*/version = \"$appversion\"/" Cargo.toml
```

```suggestion
        run: |
          appversion=$(echo "${{ steps.version-bump.outputs.new_tag }}" | sed 's/[v]//')
          sed -i -e "s/^version = .*/version = \"$appversion\"/" Cargo.toml
      - uses: actions/checkout@v4
      - run: rustup toolchain install stable --profile minimal
      - uses: actions/checkout@v4
      - run: rustup update stable && rustup default stable
      - name: regenerate cargo.lock
        run: cargo generate-lockfile
```

I would like to suggest one last minor suggestion, that is to regenerate `cargo.lock` file after the CI bumps the app version via `cargo.toml`. This is critical because if we do not regenerate the `lockfile` then the app while building will use the old `lockfile` with the new app version. This will raise errors while building the app as well as not allowing the user to build the app because of these errors.

---

Working on this now. Thanks for the suggestion!